packages/sources/cryptocompare-test/src/endpoint/crypto-ws.ts

```diff
@@ -61,17 +61,33 @@ export const wsTransport = new WebSocketTransport<WsEndpointTypes>({
       if (message.TYPE === '5' && 'PRICE' in message) {
         return [
           {
-            params: { base: message.FROMSYMBOL, quote: message.TOSYMBOL, endpoint: 'crypto-ws' },
-            value: message.PRICE as number,
+            params: { base: message.FROMSYMBOL, quote: message.TOSYMBOL, endpoint: 'crypto' },
+            response: {
+              result: message.PRICE as number,
```

Comment:
It doesn't let me add the comment above but we should maybe add a case for `PRICE` not found in the message so we can return an error.